src/User.ts

```diff
@@ -67,124 +60,26 @@ export class User {
    * @param serializedKeystore Serialized [eth-lightwallet](https://github.com/ConsenSys/eth-lightwallet) key store.
    */
   public async load (serializedKeystore: string): Promise<UserObject> {
-    const parsedKeystore = JSON.parse(serializedKeystore)
-    // check if keystore version is old and update to new version
-    if (parsedKeystore.version < 3) {
-      serializedKeystore = await this._updateKeystore(parsedKeystore)
-    }
-    const deserialized = lightwallet.keystore.deserialize(serializedKeystore)
-    const { address, keystore, pubKey } = await this._getUserObject(deserialized)
-    this.address = address
-    this.keystore = keystore
-    this.pubKey = pubKey
-    return {
-      address: this.address,
-      keystore: this.keystore.serialize(),
-      pubKey: this.pubKey
-    }
-  }
-
-  /**
-   * Takes a raw transaction and digitally signs it with the currently loaded keystore.
-   * @param rawTx RLP encoded hex string of transaction.
-   */
-  public signTx (rawTx: any): Promise<any> {
-    return new Promise((resolve, reject) => {
-      this.keystore.keyFromPassword(this._password, (err: any, pwDerivedKey: any) => {
-        if (err) {
-          return reject(err)
-        }
-        resolve(lightwallet.signing.signTx(
-          this.keystore,
-          pwDerivedKey,
-          rawTx,
-          this.address.toLowerCase() // NOTE eth-lightwallet does not handle checksum addresses
-        ))
-      })
-    })
+    const loadedAccount = await this._signer.loadAccount(serializedKeystore)
+    this.address = loadedAccount.address
+    this.keystore = loadedAccount.keystore
```

Comment:
Yes true this is eth-lightwallet only... Will remove it. But it should still be returned by `loadAccount` and `createAccount` right? Because the client needs to store it.

Reply:
Yes, as it would otherwise break old code. But maybe we should open an issue to think about changing it.